reference/motor.md

```diff
@@ -237,6 +240,21 @@ a simulation instability can appear if `position` is exactly equal to one of the
 bounds defined by the `minStop` and `maxStop` fields at the simulation startup.
 Warnings are displayed if theses rules are not respected.
 
+### Energy Consumption
+
+If the [Robot](robot.md) ancestor of a [Motor](motor.md) node has a `battery` field defined, then the energy consumption is computed for the whole robot, adding energy consumption of every device, including this motor. The energy consumption (expressed in Joule) is computed by integrating the power consumption over time (expressed in Watt). The power consumption for a rotational motor (`electrical_input_power`) is computed according to the following equation:
```

Comment:
This first sentence is quite complicated.
Maybe I would split it just after "whole robot" and explain in a second sentence how is composed the whole robot consumption.